docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -478,10 +478,44 @@ Limitations:
 5.0 Model run requests
 ++++++++++++++++++++++
 
-.. todo::
+.. list-table:: Model run plan as of October 4, 2024
+  :header-rows: 1
 
-  Fill in this section as we continue to work
+  * - Number
+    - Run
+    - Status
+    - Priority
+    - Number of draws
+    - Population size per draw
+    - Note
+  * - 1
+    - Wave I Pakistan maternal V&V
+    - Incomplete
+    - N/A
+    - 20
```

Comment:
20 was for production runs. I think we could use a better/more standardized way to decide on draws/seeds for v&v runs but half is usually what I would go for, so 10 draws for V&V.